### DISCUSSION TITLE
UI

### DISCUSSION THREAD

**Discussion by p1xelll** [advplyr/audiobookshelf-app#33] [Ideas]
**First view**

What would you like to see or change there?

[Prototype](https://www.figma.com/proto/Q0u2DFyFdC8bvM0CQRpugw/mobile?page-id=0%3A1&node-id=52%3A2852&viewport=292%2C48%2C0.58&scaling=min-zoom&starting-point-node-id=52%3A2852&show-proto-sidebar=1)
[Figma](https://www.figma.com/file/Q0u2DFyFdC8bvM0CQRpugw/mobile?node-id=0%3A1)

Added/will be added features from web app.

**Comment by advplyr** [comment]
This is great!
I'm sure I will come across things to modify as I build it out, but for now I'll start with just how you have it.

**Reply by advplyr** [reply]
One thing missing from your design is downloads. I've been really struggling with the functionality for downloads, so I'm not sure how this should look or function yet.

**Reply by p1xelll** [reply]
I had completely forgotten about the download.  :)

**Comment by Budlyte** [comment]
This looks really good!  Well done.

**Comment by p1xelll** [comment]
**Update**

Changed

- Search

Added:

- Downloads Page
- Settings Page
- Libraries Page
- Stats Page
- Users Page
- Bacup Page
- Account Page
- Upload Page

I want to keep a consistent design with the web.

**Reply by advplyr** [reply]
I just deployed a release with the start of this new design. I am trying to get the google play store to approve the app for android auto, so hopefully this will be live tomorrow. It took them 4 days to deny the last release.
Looking great so far with the nav drawer and new home/libraries tabs!

**Reply by advplyr** [reply]
Good news, google finally approved the app release. `v0.9.21-beta` has some of the new design implemented

**Comment by p1xelll** [comment]
change of top nav bar
![sub menu - icon  - home](https://user-images.githubusercontent.com/81173399/142269154-642aad77-dd2c-4f57-ae06-ef1dc2909aae.png)
![sub menu - icon - collections](https://user-images.githubusercontent.com/81173399/142269156-1a485c04-2ee1-4198-ab64-975216679179.png)
![sub menu - icon  - authors](https://user-images.githubusercontent.com/81173399/142270266-9ec4c9a4-ff91-4d46-b068-1c9b23c8896a.png)
